backend/src/main/java/com/google/models/DataInfo.java

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.models;
+
+
+/** 
+ * Wrapper for a type of data.
+ * This is an immutable class.  
+ */
+public final class DataInfo {
+  
+  private final String metricName;
+  private final String dimensionName;
+  private final String username;
+
+  /** TODO: Change username to List to deal with multiple users requesting same related data. */
```

Comment:
is this a difficult change to make? (not judging, just wondering if you wanted to do it later or doing it in this PR)

Reply:
it is not a very difficult change, but it requires changes in multiple places (ex. `RelatedData,java`, `SimpleRelatedDataGenerator.java` ...) so i think to shorten this pr, i will do that in the future in another pr.